### PR TITLE
from-alias

### PR DESCRIPTION
**Pull request by Hirobreak**
- Swaping From Alias on Email Load

### CODE REVIEW

File: email_composer/src/containers/Composer.js [hunk 1]
```diff
@@ -202,6 +204,15 @@ class ComposerWrapper extends Component {
       })
       .filter(alias => alias.isAliasActive);
     state = { ...state, accounts: [...accounts, ...aliasAccounts] };
+    if (fromAddress) {
```
Comment (danieltigse) [inline email_composer/src/containers/Composer.js:207]: You can just get fromAddress from `composerData`. You never change the value of `fromAddress`

File: email_composer/src/utils/EmailUtils.js [hunk 2]
```diff
@@ -284,13 +285,50 @@ export const formDataToReply = async (emailKeyToEdit, replyType) => {
...
+    from,
+    contacts.to,
+    replyType,
+    myEmailAddress,
+    aliases
+  );
+  const previousCcEmails = filterRecipientObject(
+    contacts.cc,
+    myEmailAddress
+  ).filter(contact => {
```
Comment (danieltigse) [inline email_composer/src/utils/EmailUtils.js:298]: include this filter inside `filterRecipientObject`

File: email_composer/src/containers/Composer.js (hunk 1 above)
Comment (Hirobreak) [reply email_composer/src/containers/Composer.js:207]: `composerData` is not in the scope

Comment (danieltigse) [reply email_composer/src/containers/Composer.js:207]: you are right

**Pull request merged**